### ISSUE TITLE
[ problem ] Chatbot did not process the pdf + got stuck

### ISSUE THREAD

**Issue by parmi02** [eidolon-ai/eidolon#851]
Description:
When using the chatbot, I uploaded a pdf (uploaded successfully). However, when I wrote a message and clicked enter, the message disappeared, and the result didn't load. It seems that the output did not go through.

1. Go to chatbot
2. Attach a pdf.
3. Write a question about the pdf.

**Comment by LukeLalor** [comment]
@parmi02 could you upload the pdf you got an error with? or was it any pdf?

**Comment by parmi02** [comment]
any pdf

**Comment by LukeLalor** [comment]
@parmi02 this works for me, can you upload your server logs and the file you attempted to upload?

```
docker compose logs > logs.txt
```

https://github.com/user-attachments/assets/a807f1bb-1e35-4523-bdc3-d61a3393224d

**Comment by parmi02** [comment]
[Eidolon -- AI Agent Server for the Enterprise.pdf](https://github.com/user-attachments/files/17438933/Eidolon.--.AI.Agent.Server.for.the.Enterprise.pdf)

This was the pdf I uploaded. Let me try again to see if the issue perists.

**Comment by flynntsang** [comment]
@flynntsang to try to repo

**Comment by flynntsang** [comment]
@parmi02 @LukeLalor I could not reproduce. I have a Macbook running Linux Mint (Ubuntu 24.04), 16GB RAM. I uploaded and conversed using my own PDFs and the attachment to this issue.

**Comment by parmi02** [comment]
It worked, wasn't able to reproduce.
